helm/charts/hydra/files/automigrate.sh

```diff
@@ -0,0 +1,40 @@
+set -e
```

Comment:
This can be removed right?

Reply:
actually IMO that's good as we want to exit if any command return non 0. 
But I would add here also `-o pipefail` as we have some pipes here

Reply:
I mean the file! ;)

Reply:
but why? since this is actual script that does the work? or you want to have it inlined in the job definition (hope not ;) )

Reply:
Maybe I don't understand what this is doing - what is the script for?

Reply:
For easier development we keep the file outside (don't need to worry about yaml indent, and can run some linters on it easier), and include it into the job here: https://github.com/ory/k8s/pull/248/files#diff-9474db6f15521172343a9f42d42a172ba3143f5d9e742727d8fdb955382b3982R33

Reply:
But I don't understand what this script is for? It is creating a config map with a JSON that has a timestamp and it is pinging the database? But why do we need that? The Hydra CLI is already doing all of that for us and it is unit, functional, and e2e tested where as this script is not which will make it hard to maintain / merge patches plus it is doing something that is already solved. Hence my suggestion to not use this approach

Reply:
Ok I think I see your point. The idea behind the CM was to not schedule hydra pods until migration was done (the deployment would wait until the CM was created). The nc role is to wait for the db to be ready, no point in keeping the pods in a crash state until the db is up and running.